reana/reana_dev/cluster.py

```diff
@@ -319,15 +317,21 @@ def job_mounts_to_config(job_mounts):
     helm_install = "cat <<EOF | helm install reana helm/reana -n {namespace} --create-namespace --wait -f -\n{values}\nEOF".format(
         namespace=namespace, values=values_dict and yaml.dump(values_dict) or "",
     )
-    cmds = [
-        "helm dep update helm/reana",
-        helm_install,
-        "kubectl config set-context --current --namespace={}".format(namespace),
-        os.path.join(
-            get_srcdir("reana"),
-            f"scripts/create-admin-user.sh {instance_name} {admin_email} {admin_password}",
-        ),
-    ]
+    cmds = []
+    if mode in ("debug"):
+        cmds.append("reana-dev git-submodule --update")
```

Comment:
In theory, we shouldn't need this second `reana-dev git-submodule --update`, however, it is the only way I could make this work.

On the one hand, it is not logical, I've done some research and I see no clear explanation of why it happens (more info [here](https://setuptools.readthedocs.io/en/latest/setuptools.html#development-mode)). On the other hand, in our use case it makes perfect sense because it covers the case of developing submodules e.g:
```
$ reana-dev run-ci --mode debug
$ # developer creates new feature with changes in R-Commons and R-Server
$ reana-dev cluster-undeploy
$ reana-dev cluster-deploy --mode debug  # thanks to the debug flag, the new R-Commons will be fetched and updated
```